lib/v2/baozimh/index.js

```diff
@@ -33,7 +33,15 @@ module.exports = async (ctx) => {
 
                 const detailResponse = await got.get(descUrl);
                 const $ = cheerio.load(detailResponse.data);
-                const desc = $('div.comic-contain > amp-img > noscript').text();
+                const desc = $('section.comic-contain')
+                    .find('amp-img')
+                    .map((_, item) => {
+                        const imgUrl = $(item).attr('src');
+
+                        return `<img src="${imgUrl}">`;
```

Comment:
Please use [art-template](https://docs.rsshub.app/joinus/script-standard.html#template) to render HTML. https://github.com/DIYgod/RSSHub/discussions/8359